package.yaml

```diff
@@ -85,7 +85,9 @@ tests:
     main: README.lhs
     ghc-options: -pgmL markdown-unlit
     dependencies:
-      - Blammo
       - aeson
+      - Blammo
```

Comment:
There may be a slight locale difference here, as you seem to have sorted a-then-B whereas our `sort` (at least mine) did B-then-a. Do you mind putting this back to avoid any back and forth between developers? I'm not sure what's "correct" but capitals-first is what we conventionally do.

Reply:
Ah indeed, I missed that. Seems to be VS Code's "Sort Lines Ascending" preference. Will put it back no problem.

Reply:
Restyled also failed with a pretty big diff on the `Blammo.cabal` file, I was a little surprised. Do you want me to commit that as well?

![image](https://github.com/freckle/blammo/assets/1306536/4b634b03-aff0-4a25-8e01-a35180837e96)